infra/aws/terraform/kops-infra-ci/eks.tf

```diff
@@ -17,14 +17,18 @@ limitations under the License.
 module "eks" {
   providers = { aws = aws.kops-infra-ci }
   source    = "terraform-aws-modules/eks/aws"
-  version   = "19.16.0"
+  version   = "20.8.3"
 
   cluster_name                   = local.cluster_name
   cluster_version                = var.eks_version
   cluster_endpoint_public_access = true
 
   cluster_ip_family = "ipv4"
 
+  # Give the Terraform identity admin access to the cluster
+  # which will allow resources to be deployed into the cluster
+  enable_cluster_creator_admin_permissions = true
```

Comment:
@bryantbiggs Do we still need the kubernetes provider with this ?

Reply:
no - you only need the Kubernetes provider if you will continue to create `aws-auth` entries using the (new) [sub-module](https://github.com/terraform-aws-modules/terraform-aws-eks/tree/master/modules/aws-auth). This was the only area where the Kubernetes provider was used

Reply:
I still have a few [resources](https://github.com/kubernetes/k8s.io/blob/main/infra/aws/terraform/kops-infra-ci/metrics-server.tf) created by the k8s provider. So If understand correctly, those resources can be actuated using the access entries ?

Reply:
I don't see any resources at that link that could be replaced by access entry. In short:
- Access entry allows IAM principals (roles and users) access to Kubernetes resources on the cluster
- Pod identity maps an IAM role to a service account in a namespace in the specified cluster